src/ape/_cli/__init__.py

```diff
@@ -0,0 +1,32 @@
+import click
+from ape_compile._cli import cli as compile
+from ape_console._cli import cli as console
+from ape_networks._cli import cli as networks
+from ape_plugins._cli import cli as plugins
+
+from ape._cli.groups import ApeCLI
+from ape._cli.options import shared_options
+from ape_accounts._cli import cli as accounts  # type: ignore
+
+CONTEXT_SETTINGS = {
+    "help_option_names": ["-h", "--help"],
+    "max_content_width": 200,
+}
+
+
+@click.group(
+    cls=ApeCLI,
+    context_settings=CONTEXT_SETTINGS,
+    invoke_without_command=True,
+    no_args_is_help=True,
+)
+@shared_options()
+def cli():
+    ...
```

Comment:
`pass`?

Reply:
Yeah, I can use `pass`

Reply:
The reason I used `...` instead is because I saw it somewhere else in the code base I am pretty sure, and it's the same thing, right?

Reply:
I used it with abstract base classes, I think `abstractmethod` uses `...` to determine abstract methods somehow

Reply:
```suggestion
    pass
```